lib/cog_cmd/consul/consul.rb

```diff
@@ -13,4 +13,16 @@ def domain
   def consul_token
     env_var("CONSUL_TOKEN", required: true)
   end
+
+  def permitted_channels
+    env_var("CONSUL_CHANNELS")
```

Comment:
@loganmeetsworld Probably want to parse this as a comma-delimited list:

```ruby
env_var('CONSUL_CHANNELS').split(/,\s*/)
```

As written, `permitted_channels.include?(current_channel)` would match any room that's a substring of `permitted_channels`.